public/app/features/integrated-alerting/components/AlertRuleTemplate/AlertRuleTemplatesTable/AlertRuleTemplatesTable.tsx

```diff
@@ -47,6 +48,11 @@ export const AlertRuleTemplatesTable = () => {
         accessor: 'created_at',
         width: '10%',
       } as Column,
+      {
+        Header: actionsColumn,
+        accessor: (template: FormattedTemplate) => <AlertRuleTemplateActions template={template} />,
+        width: '10%',
```

Comment:
nit: relative width makes the action column too wide or too marrow sometimes, depending on the screen size. Is there a way to define a fixed width for the actions column?

Reply:
That's right!! I tried a fixed value here but it doesn't work (seems like the width of cells is getting calculated afterwards). So I'm investigating how to do it with react-table.

Reply:
Can we pass something like `maxWidth: 200`?

Reply:
We can but doesn't work either when using percentage in the other columns. But setting all the other columns percentage to add up 100% did the trick. So now the actions column width is fixed!